src/runtime_src/core/tools/common/tests/TestBandwidthKernel.cpp

```diff
@@ -309,6 +308,7 @@ TestBandwidthKernel::runTest(std::shared_ptr<xrt_core::device> dev, boost::prope
       std::vector <double> throughput_per_kernel = throughputs.second;
       logger(ptree, "Details", boost::str(boost::format("Throughput (Type: DDR) (Bank count: %d) : %.1f MB/s") % num_kernel_ddr % max_throughput));
       for (int i = 0; i < num_kernel_ddr; i++)
+        if(bank_names.size() > 0 && throughput_per_kernel.size() > 0)
```

Comment:
please fix this logic.

#use below code
if (bank_names.size() == num_kernel_ddr && throughput_per_kernel.size() == num_kernel_ddr) {
   for (int i = 0; i < num_kernel_ddr; i++)
          logger(ptree, "Details", boost::str(boost::format("Throughput of Memory Tag: %s : %.1f MB/s") % bank_names[i] %    throughput_per_kernel[i]));
}

Reply:
Hi @chvamshi-xilinx fixed the logic with latest commit.